play-scala-tls-example/build.sbt

```diff
@@ -25,3 +25,7 @@ fork in run := true
 javaOptions in run += "-Djavax.net.debug=ssl:handshake"
 
 addCommandAlias("client", "runMain Main")
+
+// Must not run tests in fork because the `play` script sets
+// some JVM properties (-D) which tests need.
+fork in Test := false
```

Comment:
Can we do the opposite and move everything the the `play` script into the build and drop the need for a custom `play` script?

Reply:
I wouldn't know where to start. We can try to do it on a separate PR but I prefer this complexity to live on a bash script rather than an `sbt` script.

Reply:
Sure, let's consider it in another PR.

But I feel the opposite: I'd rather not have this complexity require remembering not to call plain `sbt` but to use the specialised `play` bash script.  These are settings related to how the code is tested, which is within the realms of build system configuration.

Reply:
Btw, I didn't mean to imply in this PR.  I was just asking because it seemed you'd understand this part of the setup so I was looking for insight.